src/Octoshift/GithubClient.cs

```diff
@@ -29,6 +31,8 @@ public GithubClient(OctoLogger log, HttpClient httpClient, string personalAccess
             }
         }
 
+        public virtual async Task<string> GetNonSuccessAsync(string url, HttpStatusCode status) => (await SendAsync(HttpMethod.Get, url, status: status)).Content;
```

Comment:
Again I still stand by my original comment and think that this is a bad design in a lower level client. the client should be agnostic and do just the basic stuff. If some API returns 3XX as a success code then we should just accommodate it in our business (ie. `GithubApi`)